types/robust-segment-intersect/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for robust-segment-intersect 1.0
+// Project: https://github.com/mikolalysenko/robust-segment-intersect
+// Definitions by: masx200 <https://github.com/masx200>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type Coordinate = [number, number];
+declare function a (
+        a0: Coordinate,
+        a1: Coordinate,
+        b0: Coordinate,
+        b1: Coordinate
+    ) :boolean;
+export default a;
```

Comment:
This line should probably be `export =` instead of `export default`, since the original library uses `module.exports`: <https://github.com/mikolalysenko/robust-segment-intersect/blob/cbf20e2abbb22bda5b7919823f58c856ab6ac403/segseg.js#L3>

```suggestion
type Coordinate = [number, number];
declare function a (
        a0: Coordinate,
        a1: Coordinate,
        b0: Coordinate,
        b1: Coordinate
    ) :boolean;
export = a;
```